compose/profiles.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Using profiles with Compose
+desription: Using profiles with Compose
+keywords: cli, compose, profile, profiles reference
+---
+
+Profiles allow to adjust the Compose application model for various usages and
+environments by selectively enabling services. For this you can assign a
+service to profiles under which the service should be started only.
```

Comment:
```suggestion
This is achieved by assigning each service to zero or more profiles. If unassigned, the service is _always_ started but if assigned, it is only started if the profile is activated.
```

Reply:
Not 100 % sure about this change. A little worried that I'm repeating what's said lower down.